utils.py

```diff
@@ -59,7 +61,13 @@ def to_str(value: str, default: Optional[str] = None) -> Optional[str]:
 
 def to_text(value: str) -> Optional[str]:
     """Function ensures proper encoding of entities"""
-    return replace_text(value, TEXT_REPLACEMENTS)
+    if value:
+        value = replace_text(value, TEXT_REPLACEMENTS)
+        # also ensure proper line endings
+        for match in LINE_ENDINGS.finditer(value):
+            ending = match.group(0)
+            value = value.replace(ending, "\n\n")
```

Comment:
Consider to use [``re.sub``](https://docs.python.org/3/library/re.html#re.sub) to do this all at once.

Reply:
I updated the code and also included code from another PR (#30) as this could be similarly improved this way.